Engine/Results/BacktestingResultHandler.cs

```diff
@@ -767,7 +767,11 @@ private string ProcessLogMessages(AlgorithmNodePacket job)
         {
             if (_exitTriggered)
             {
-                _api.StoreSystemLogs(job);    
+                // Process the system logs once exit has been triggered. This should occur the second time this method is called
+                if (!_api.StoreSystemLogs(job))
+                {
+                    Log.Error("BacktestingResultsHandler.ProcessLogMessages(): Could not upload the system logs to S3.");
```

Comment:
Wouldn't this logging be more appropriate inside the api implementation? This way, the error message would be specific to the implementation vs the logging assuming an S3 implementation